.storybook/theme/utils/global-theme-controller.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {Fragment, useEffect} from 'react';
+import {Theme, useTheme} from '../../../src';
+
+export interface GlobalThemeControllerProps {
+    children?: React.ReactNode;
+}
+
+export const GlobalThemeController = ({children}: GlobalThemeControllerProps) => {
+    const theme = useTheme();
+    const [prevTheme, setPrevTheme] = React.useState(theme);
+
+    const updateBodyClassName = (newTheme: Theme) => {
+        const bodyEl = document.body;
+
+        if (!bodyEl.classList.contains('yc-root')) {
+            bodyEl.classList.add('yc-root');
+        }
+
+        bodyEl.classList.toggle('yc-root_theme_light', newTheme === Theme.Light);
```

Comment:
with uikit5 `yc-root` becomes `g-root`
also let's create a constant for this class